src/clear.js

```diff
@@ -1,19 +1,19 @@
-/* eslint-disable */
-import setStorage from "./storage.js";
-import { deleteTask } from "./crud-methods.js";
-
+import setStorage from './storage.js';
+import { deleteTask } from './crud-methods.js';
+// eslint-disable-next-line import/prefer-default-export, no-use-before-define
 export { clearAll };
 
 const clearAll = () => {
-  const localData = JSON.parse(localStorage.getItem("todo"));
-  const taskContainer = document.querySelectorAll(".list-item");
+  const localData = JSON.parse(localStorage.getItem('todo'));
+  const taskContainer = document.querySelectorAll('.list-item');
   taskContainer.forEach((e) => {
-    if (e.classList.contains("list-selected-item")) {
+    if (e.classList.contains('list-selected-item')) {
       deleteTask(e);
     }
   });
   let count = 0;
   const data = Array.from(localData).filter((e) => e.completed === false);
+  // eslint-disable-next-line no-return-assign, no-multi-assign
   data.map((e) => (e.index = count += 1));
   setStorage(data);
 };
```

Comment:
- [ ] Dear @Cabraham1, I notice you have removed some of ignored linter tules as requested by the previous reviewer. But while doing that I also notice you have disabled `linters` somewhere else too. Remember `linters` are here to help us write clean code and we should use them accordingly. Please remove all linter-disabled lines.